app/lib/features/news/pages/add_news_page.dart

```diff
@@ -141,15 +190,12 @@ class AddNewsState extends ConsumerState<AddNewsPage> {
 
   //Action Button
   Widget actionButtonUI(BuildContext context) {
-    return Visibility(
```

Comment:
Does this mean it now validates without having singular boost to post???

Reply:
no, this if just has been moved to the outside caller rather than render not animated invisible elements - https://github.com/acterglobal/a3/pull/2271/files/486be7ffe7f856e992d00b5a1f5ab757d9bafbdb#diff-54bc38247b32ff741c73e257f09a65d854d5d6c7f02ae92a24b08f282f86b5b1R99